gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Nov 14 17:00:44 EET 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
I suggest to use the latest version of gradle 
https\://services.gradle.org/distributions/gradle-6.0.1-bin.zip
also this version is generated by latest version of jhipster generator